storage/client/client.go

```diff
@@ -277,6 +283,21 @@ func (c *StorageClient) Blocks(ctx context.Context, r apiTypes.GetConsensusBlock
 	return &bs, nil
 }
 
+func validateHash(input string) (string, error) {
```

Comment:
```suggestion
func canonicalizedHash(input string) (string, error) {
```

or `dbFormattedHash()` or `normalizedHash()` or sth.

A `validate*` name implies a bool return to me.